src/crimson/os/seastore/seastore_types.h

```diff
@@ -1738,17 +1738,27 @@ struct __attribute__((packed)) object_data_le_t {
   }
 };
 
+enum class omap_type_t : uint8_t {
+  XATTR = 0,
+  OMAP_LARGE_LEAF,
+  OMAP_SMALL_LEAF,
```

Comment:
Given the new ObjectStore semantic, probably `LOG` is a better name. Like `XATTR` and `OMAP`, it describes the type of metadata stored.